src/commands/lint.cr

```diff
@@ -0,0 +1,77 @@
+module Mint
+  class Cli < Admiral::Command
+    class Lint < Admiral::Command
+      include Command
+
+      define_help description: "Lints the project"
+
+      define_flag json : Bool,
+        description: "Output errors to a JSON file",
+        default: false,
+        required: false
+
+      define_flag output : String,
+        description: "The output filename",
+        default: "lint.json",
+        required: false,
+        short: "o"
+
+      def run
+        execute "Linting" do
+          lint
+        end
+      end
+
+      def lint
+        json =
+          MintJson.parse_current
```

Comment:
The errors from the `mint.json` can be catched like so:
```suggestion
          begin
            MintJson.parse_current
          rescue ex
            ex_handler errors, ex
          end
```

Reply:
This seemed to be getting overridden by:

```crystal
sources =
  Dir.glob(SourceFiles.all)
```

So I removed `MintJson.parse_current` and wrapped a `begin...rescue` around this instead